Makefile

```diff
@@ -224,12 +224,17 @@ deploy-test: generate-keys
 	&& export DUMMY_IGNITION=${DUMMY_IGNITION} && ISO_CREATION=minikube-local-registry/assisted-iso-create:minikube-test \
 	$(MAKE) update-minikube deploy-wiremock deploy-all
 
-deploy-onprem:
+deploy-onprem-base:
 	podman pod create --name assisted-installer -p 5432,8000,8090,8080
 	podman run -dt --pod assisted-installer --env-file onprem-environment --name db quay.io/ocpmetal/postgresql-12-centos7
-	podman run -dt --pod assisted-installer --env-file onprem-environment --user assisted-installer  --restart always --name installer $(SERVICE_ONPREM)
 	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always -v $(PWD)/deploy/ui/nginx.conf:/opt/bitnami/nginx/conf/server_blocks/nginx.conf:z --name ui quay.io/ocpmetal/ocp-metal-ui:latest
 
+deploy-onprem: deploy-onprem-base
+	podman run -dt --pod assisted-installer --env-file onprem-environment --user assisted-installer  --restart always --name installer $(SERVICE_ONPREM)
+
+deploy-onprem-for-subsystem: deploy-onprem-base
```

Comment:
Try to see if `deploy-on-prem-for-subsystem` can use `deploy-onprem` to avoid a duplication of most of the content.
(Then something would get changed one place and not in the other one..)

Reply:
Done.